### PR TITLE
Sonarcloud code smell fixes

### PR DESCRIPTION
**Pull request by MadsThy**
## Description

Small batch of code smell fixes listed on Sonarcloud

For a reference point (since sonarcloud does not provide a comparison when fixing things), please visit this page:

https://sonarcloud.io/summary/overall?id=geh-frontend-app

You can click on the code smell list (currently 30), and then search for the specific file you are looking up for review :)

I think the worst one that was changed was changing [0-9] to \d in the regEx. The rest should be straight forward

Also updated the sonarcloud version we use in our actions, since it was complaining that the action version we used was based on nodeJS 14, and they are ending support for it May 1st :)

### CODE REVIEW

File: libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts [hunk 1]
```diff
@@ -14,30 +14,30 @@
...
 import { WattSpinnerModule } from '@energinet-datahub/watt/spinner';
-import { CommonModule } from '@angular/common';
-import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
-import { MatDividerModule } from '@angular/material/divider';
-import { WattEmptyStateModule } from '@energinet-datahub/watt/empty-state';
+import { TranslocoModule } from '@ngneat/transloco';
+import { LetModule } from '@rx-angular/template/let';
+import { PushModule } from '@rx-angular/template/push';
+import { Subject, take, takeUntil } from 'rxjs';
+import { DhChargesPricesDrawerService } from '../dh-charges-prices-drawer.service';
```
Comment (dzhavat) [inline libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts:40]: question: do you have an extension that did this? Looks like the imports are sorted alphabetically :)

File: libs/dh/shared/assets/src/assets/b2c-custom-layouts/login-unified.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (dzhavat) [inline libs/dh/shared/assets/src/assets/b2c-custom-layouts/login-unified.html:18]: suggestion: the changes are primarily formatting (except this one) but it's probably still a good idea to upload the file after merge. What do you think @frontend-specialisten :)

File: libs/dh/shared/assets/src/assets/b2c-custom-layouts/self-asserted.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (dzhavat) [inline libs/dh/shared/assets/src/assets/b2c-custom-layouts/self-asserted.html:18]: suggestion: the changes are primarily formatting (except this one) but it's probably still a good idea to upload the file after merge. What do you think @frontend-specialisten :)

File: .github/workflows/frontend-ci.yml [hunk 4]
```diff
@@ -100,7 +100,7 @@ jobs:
         with:
           fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
       - name: SonarCloud Scan
-        uses: SonarSource/sonarcloud-github-action@v1.6
+        uses: SonarSource/sonarcloud-github-action@master
```
Comment (dzhavat) [inline .github/workflows/frontend-ci.yml:103]: Living on the edge 🚀 🙂

File: libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts (hunk 1 above)
Comment (MadsThy) [reply libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts:40]: This is done by VSCode with this setting in my settings.json:

` 
"editor.codeActionsOnSave": {
    "source.organizeImports": true,
    "source.sortMembers": true
  }
`

File: .github/workflows/frontend-ci.yml (hunk 4 above)
Comment (MadsThy) [reply .github/workflows/frontend-ci.yml:103]: Gotta go fast 💨 💨 🕺 ⏩

File: libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts (hunk 1 above)
Comment (dzhavat) [reply libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts:40]: I see. Also a good solution. So it looks like everyone has their own way of organizing imports :D 

Some just add them as they come, without order, others put them in groups, you let VS Code do it, ...  

Not sure what a common solution would look like, but that shouldn't stop the PR from merging.

**Comment by dzhavat** [comment]
Hey @MadsThy 
Can we get this to `main`? I can help with the conflicts if you don't have time :)

File: libs/dh/shared/assets/src/assets/b2c-custom-layouts/login-unified.html (hunk 2 above)
Comment (dzhavat) [reply libs/dh/shared/assets/src/assets/b2c-custom-layouts/login-unified.html:18]: Figured that the `lang` attribute is added automatically when visiting the page, so it's really not necessary to upload the files again. So it seems that this change is mainly to please SonarCloud. But it's a good change nonetheless ;)

**Review by dzhavat — APPROVED** [review]

**Pull request merged**